node/engine/chainservice/eth_chainservice.go

```diff
@@ -344,18 +341,16 @@ out:
 
 		case err := <-eventSub.Err():
 			if err != nil {
```

Comment:
This changes here means the chainservice will attempt to re-establish the websocket connection once before triggering a panic. We can come back later and refine this code if we want (e.g. add exponential back-off and ensure no missed chain events between websocket error and resubscribing)

Reply:
It could be nice to add a comment with some of that info to the code; it took me a second to understand this was the retry logic.

Reply:
Added comment in https://github.com/statechannels/go-nitro/pull/1660/commits/56607ea190829746e6c0531e435ca948cd131e4e